tron/bin/check_tron_jobs.py

```diff
@@ -52,6 +53,13 @@ def parse_cli():
         dest="run_interval",
         default=300,
     )
+    parser.add_argument(
+        "--skip-failed-logging-for-superregion",
+        help="Boolean flag to skip querying stoud/stderr logs for failed jobs for the current superregion",
```

Comment:
nit (typo + some re-wording):
```suggestion
        help="Skip including stdout/stderr logs in alerts for failed jobs",
```